File: ProyectoFinal/ProyectoFinal/2_Dominio/Entidades/Taxonomia.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace ProyectoFinal._2_Dominio.Entidades
{
    public class Taxonomia
    {
        public int IdTaxonomia { get; set; }

        public string Nombre { get; set; }

        public bool? Activo { get; set; }
    }
}

File: Interfaz/ProyectoFinal/src/app/Components/Taxonomia/class/taxonomia.ts
export class Taxonomia{
    private idTaxonomia!: number;
    private nombre!: string;

    constructor(){}

    public get IdTaxonomia(): number {
        return this.idTaxonomia;
    }
    public set IdTaxonomia(value: number) {
        this.idTaxonomia = value;
    }
    public get Nombre(): string {
        return this.nombre;
    }
    public set Nombre(value: string) {
        this.nombre = value.trim();
    }
}